update-covid-19.py

```diff
@@ -43,9 +43,12 @@ def main():
 
 def write_image():
     df = pandas.read_csv(CSV_FILE)
-    rcParams.update({'figure.autolayout': True})
-    fig = df.plot('Date', secondary_y=('New cases', 'New deaths', 'Cumulative deaths'), rot=30).get_figure()
+    matplotlib.rcParams.update({'figure.autolayout': True})
+    fig = df.plot('Date', secondary_y=('New cases', 'New deaths', 'Cumulative deaths'), rot=30, title='Linear Scale').get_figure()
     fig.savefig('covid-19-fig.png')
+    fig = df.plot(logy=True, title='Log Scale')
```

Comment:
Can you rename these variables to `fig1` and `fig2` so that there is less ambiguity and so that it's clearer that there is no sharing of information between them?